xml/schema/reviews.xml

```diff
@@ -53,7 +53,16 @@
 		</field>
 		<field name="reviewer_file_id" type="I8"/>
 		<field name="date_rated" type="T"/>
-		<field name="date_reminded" type="T"/>
+        <field name="date_invite_reminded" type="T"/>
```

Comment:
The old XML schema is only used for legacy parts of the upgrade process and shouldn't need to be modified. For anything post-3.3.0, the migrations classes take care of it.